### PR TITLE
[Fixes #64] CreateRouteMiddleware Bug

### LINKED ISSUE #64: Create Handler Issue
```javascript
const createRoute = (features, { requiredFeature, ...methods }) => (req, res, next) => {
  const parsedUrl = parse(req.url, true);
  const { search } = parsedUrl;
  const updatedFeatures = updateFeatures(features, search);
  setStatus(res, getIsEnabled(updatedFeatures, requiredFeature));

  const handler = methods[req.method.toLowerCase()];
  if (handler !== undefined && typeof handler === 'function') {
    handler(req,res);
  }

  next();
};
```

`next()` being called after the handler is called creates an issue were the status gets overridden by a future handler

`next()` should not be called if the handler is called.

### PR DESCRIPTION
**Pull request by kennylavender**
resolves #64 
Fixes `createRouteMiddleware` so that it does not call `next()` if  a method handler has been called.

### CODE REVIEW

**Comment by kennylavender** [comment]
@ericelliott I wonder if instead of mocking express it would be more efficient and accurate to install express and supertest and then create some solid integration tests? Would that be a better solution or are their reasons not to do such a thing?

**Comment by ericelliott** [comment]
Yes, integration tests are great!

**Comment by kennylavender** [comment]
- Added express and supertest to integration test this function instead of mocking.
- Deleted of mocking unit tests
- Fixed issue with package-lock.json installing a different version of tape for riteway
  - This caused riteway tests to run in a separate tape instance so failing tests would not cause the tests to actually fail.

**Comment by kennylavender** [comment]
@ericelliott ready for another review.

File: .babelrc [hunk 1]
```diff
@@ -12,6 +12,7 @@
       }
     ],
     "react",
-    "stage-2"
+    "stage-2",
+    "stage-0"
```
Comment (ericelliott) [inline .babelrc]: What stage-0 features are we using?

File: src/test/integration/context.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import describe from 'tape';
```
Comment (ericelliott) [inline src/test/integration/context.js]: Why not use riteway?

Comment (kennylavender) [reply src/test/integration/context.js]: that was an old test file, just moved the file

File: .babelrc (hunk 1 above)
Comment (kennylavender) [reply .babelrc]: I thought it was required for async/await, maybe its just the babel polyfill I needed

Comment (kennylavender) [reply .babelrc]: removed it, wasn't needed.

File: src/test/integration/context.js (hunk 2 above)
Comment (kennylavender) [reply src/test/integration/context.js]: I used `git mv` but it must have lost the connection.

**Comment by ericelliott** [comment]
Published as @paralleldrive/react-feature-toggles@1.2.0

**Pull request merged**